### PR TITLE
Added kubecontext parameter for create ns with WI and install helm release

### PR DESCRIPTION
**Pull request by tomich87**
Allow set kubecontext to migration to dev-02 gke cluster.

### CODE REVIEW

File: k8s/k8s-namespace.bzl [hunk 1]
```diff
@@ -39,8 +39,12 @@ def _k8s_namespace_impl(ctx):
         if workload_identity_namespace == "":
              fail(msg='ERROR: workload_identity_namespace must be provided if gcp_sa is set')
 
-    stamp_files = [ctx.info_file, ctx.version_file]
+    if ctx.attr.gcp_gke_project[7:] == 'dev-02':
+      kubecontext = "gke_mm-k8s-dev-02_europe-west2_mm-k8s-dev-02"
+    else:
+      kubecontext = "gke_mm-k8s-dev-01_europe-west1_mm-k8s-dev-01"
```
Comment (chrissgyulev) [inline k8s/k8s-namespace.bzl:45]: This part is a little bit crappy

Comment (tomich87) [reply k8s/k8s-namespace.bzl]: solved

**Review by chrissgyulev — APPROVED** [review]
LGTM

File: examples/base-package/BUILD [hunk 2]
```diff
@@ -19,4 +19,5 @@ helm_release(
   chart = ":base_package",
   namespace = "examples",
   release_name = "base-release",
+  kubernetes_context = "gke_mm-k8s-dev-02_europe-west2_mm-k8s-dev-02"
```
Comment (danigar) [inline examples/base-package/BUILD:22]: Maybe we should use other context name in these examples?

File: helm/helm-release.bzl [hunk 3]
```diff
@@ -102,6 +101,7 @@ helm_release = rule(
       "secrets_yaml": attr.label_list(allow_files = True, mandatory = False),
       "sops_yaml": attr.label(allow_single_file = True, mandatory = False),
       "helm_version": attr.string(mandatory = False),
+      "kubernetes_context": attr.string(mandatory = False, default = "gke_mm-k8s-dev-01_europe-west1_mm-k8s-dev-01"),
```
Comment (danigar) [inline helm/helm-release.bzl:104]: Same here. Or make this mandatory maybe?

**Review by danigar — CHANGES_REQUESTED** [review]
Minnor comments. We should add this new flags to the docs -> README.md

**Comment by danigar** [comment]
@gutmox the rule works well without this new parameters as it takes the default kube context of the jenkins node. If you are changing the cluster in the kubecontext (in the CI), should be enough.

**Review by danigar — CHANGES_REQUESTED** [review]
just add some doc to the readme.md, but everythingelse 👌

**Pull request merged**